src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingsResponseMapper.java

```diff
@@ -31,36 +31,34 @@ private void setCaseHearings(List<CaseHearingRequestEntity> entities, GetHearing
         List<CaseHearing> caseHearingList = new ArrayList<>();
         for (CaseHearingRequestEntity entity : entities) {
             CaseHearing caseHearing = getCaseHearing(entity);
-            List<HearingResponseEntity> hearingResponses = getHearingResponseEntities(entity, caseHearing);
-            setHearingDaySchedule(caseHearingList, caseHearing, hearingResponses);
+            HearingResponseEntity latestHearingResponse = getHearingResponseEntities(entity, caseHearing);
```

Comment:
Spec only wants details of the latest one, not all. Logic in this class has been changed to base it on the latest.